src/Persistence/Array_.php

```diff
@@ -119,11 +119,22 @@ public function tryLoad(Model $model, $id, string $table = null): ?array
         return $this->typecastLoadRow($model, $this->data[$table][$id]);
     }
 
+    /**
+     * Tries to load first available record and return data record.
+     */
+    public function loadAny(Model $model, string $table = null): ?array
+    {
+        $row = $this->tryLoadAny($model, $table);
+        if (!$row) {
```

Comment:
you should use strict type here

Reply:
Copy-pasted from SQL persistence. You are right, `$row === null` is more explicit.

Reply:
put there `$row === null`